### DISCUSSION TITLE
its about business hours with bot

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#4861] [Self hosted]
How can bot understand or can we code to identify set business hours from panel so that we can 
customize the bot reply

**Comment by sojan-official** [comment]
You will have to make an API call in your bot server to fetch the business hour configuration of the inbox.

**Reply by NitinNR** [reply]
I am using the dialogflow here and attach the webhook in dialogflow console and at wehbook site i manualy(hard coded)checking the business hours not fetching business hours data from the chatwoot server and returning the payload accordingly.so do we have other way to get the data whenever its change from panel like webhook call from panel to our endpoint or something like that and is it really possible with non-api inbox to call the api for the data? and can we really get the business hours through api ?
